db/primary_migrate/20220808140030_add_issuer_to_in_person_enrollments.rb

```diff
@@ -0,0 +1,8 @@
+class AddIssuerToInPersonEnrollments < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def change
+    add_column :in_person_enrollments, :issuer, :string, null: true, comment: "Issuer associated with the enrollment at time of creation"
+    add_foreign_key :in_person_enrollments, :service_providers, column: :issuer, primary_key: :issuer, validate: false
```

Comment:
Strong Migrations was pretty particular about this new foreign key, and enforced the `validate: false`. Not 100% clear if we need to enable that constraint validation, and if so whether it should happen in a separate deployment, or if it could be included here.

https://github.com/ankane/strong_migrations#good---postgres

Reply:
Is this a real foreign key constraint? or just a suggestion? (my preference is for "just a suggestion" because FK constraints can implicitly add a few joins to table operations which can be surprising performance hits)

Reply:
I'll plan to add the `validate` in a separate migration.

Reply:
Follow-up at #6734.